iceoryx_posh/test/integrationtests/test_roudi_findservice.cpp

```diff
@@ -358,7 +386,7 @@ TEST_F(RoudiFindService_test, InterfacePort)
         auto caproMessage = maybeCaProMessage.value();
         if ((caproMessage.m_serviceDescription.getServiceIDString() == IdString_t("service1"))
             && (caproMessage.m_serviceDescription.getInstanceIDString() == IdString_t("instance1"))
-            && ((caproMessage.m_serviceDescription.getEventIDString() == IdString_t(iox::capro::AnyEventString))))
+            && ((caproMessage.m_serviceDescription.getEventIDString() == IdString_t(iox::capro::InvalidString))))
```

Comment:
Was this change intentional? I guess this changes with #854?

Reply:
Yes, to both questions. Otherwise the test would fail.